File: trail1/특정 수 맞히기/catching-specific-number.java
import java.util.Scanner;
public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        while (true) {
            int N = sc.nextInt();
            if (N < 25) {System.out.println("Higher");}
            else if (N > 25) {System.out.println("Lower");}
            else {
                System.out.println("Good");
                break;
            }
        }
    }
}

File: trail1/특정 수 맞히기/catching-specific-number.c
#include <stdio.h>

int main() {
    while (1) {
        int N;
        scanf("%d", &N);
        if (N < 25) {printf("Higher\n");}
        else if (N > 25) {printf("Lower\n");}
        else {
            printf("Good\n");
            break;
        }
    }
    return 0;
}